@navikt/core/css/darkside/accordion.darkside.css

```
@@ -115,67 +115,55 @@
 }
 
 /* ---------------------------- Accordion content --------------------------- */
-.navds-accordion__content {
-  --__acx-accordion-content-line-width: 2px;
 
-  padding-inline: var(--ax-space-24) var(--ax-space-12);
-  display: none;
-  overflow: hidden;
-  padding-block: 0;
-  max-height: 0;
-  border-color: transparent;
-  transition: all 250ms cubic-bezier(0.2, 0, 0, 1) allow-discrete;
-  animation: navds-accordion-content-animation 250ms cubic-bezier(0.2, 0, 0, 1);
-}
+.navds-accordion--indent > .navds-accordion__item .navds-accordion__content-inner {
+  padding-block: var(--ax-space-8);
+  padding-inline: calc(var(--ax-space-20) + 2px) var(--ax-space-20);
 
-.navds-accordion__item--no-animation {
-  animation: none;
+  @media (forced-colors: active) {
+    border-left: 1px solid canvastext;
+  }
 }
 
-@keyframes navds-accordion-content-animation {
-  0% {
-    opacity: 0.01;
-  }
+.navds-accordion__item {
+  & > .navds-accordion__content {
+    display: grid;
+    box-shadow: -2px 0 0 0 var(--ax-border-subtleA);
+    grid-template-rows: 0fr;
+    visibility: hidden;
+    overflow: hidden;
+    margin-inline: var(--ax-space-24) var(--ax-space-12);
+    margin-block: 0;
+    transition-timing-function: cubic-bezier(0.2, 0, 0, 1);
+    transition-duration: 250ms;
+    transition-property: visibility, opacity, height, grid-template-rows, margin, min-height;
```

Don't see height doing anything here
```suggestion
    transition-property: visibility, opacity, grid-template-rows, margin, min-height;
```